lib/types/types.ts

```diff
@@ -95,3 +95,27 @@ export type committeeInterviewType = {
   timeslot: string;
   message: string;
 };
+
+export type interviewType = {
```

Comment:
kanskje intervjutypen burde ha komiténavn og navn på søker?

Reply:
interviewType.name blir navnet på søkeren eller komiteen

Reply:
er det ikke lurere å ha hver sin attribut for navnene